src/main/java/step2/RacingGame.java

```diff
@@ -0,0 +1,55 @@
+package step2;
+import java.util.Random;
+
+public class RacingGame {
+
+    private int time;
+    private int[] carPositions;
+    private ResultView resultView;
+    public RacingGame() {
+        this.resultView = new ResultView();
+    }
+
+    public int[] carPositionsInitiate(int createCarCount) {
+        this.carPositions = new int[createCarCount];
+        for (int i = 0; i < carPositions.length; i++) {
+            carPositions[i] = 0;
+        }
+        return carPositions;
+    }
+
+    public int setTime(int time) {
+        this.time = time;
+        return time;
+    }
+
+    public int getRandomNumber() {
```

Comment:
`RacingGame` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private int getRandomNumber() {
```

Reply:
넵 감사합니다!!